MapboxNavigation/NavigationMapView.swift

```diff
@@ -162,30 +159,34 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
                 enableFrameByFrameCourseViewTracking(for: 3)
                 altitude = defaultAltitude
                 showsUserLocation = true
-                courseTrackingDelegate?.navigationMapViewDidStartTrackingCourse?(self)
+                courseTrackingDelegate?.navigationMapViewDidStartTrackingCourse(self)
             } else {
-                courseTrackingDelegate?.navigationMapViewDidStopTrackingCourse?(self)
+                courseTrackingDelegate?.navigationMapViewDidStopTrackingCourse(self)
             }
             if let location = userLocationForCourseTracking {
                 updateCourseTracking(location: location, animated: true)
             }
         }
     }
 
+
     /**
-     A `UIView` used to indicate the user’s location and course on the map.
+     A type that represents a `UIView` that is `CourseUpdatable`.
+    */
+    public typealias UserCourseView = UIView & CourseUpdatable
+    
+    /**
+     A `UserCourseView` used to indicate the user’s location and course on the map.
      
-     If the view conforms to `UserCourseView`, its `UserCourseView.update(location:pitch:direction:animated:)` method is frequently called to ensure that its visual appearance matches the map’s camera.
+     The `UserCourseView`'s `UserCourseView.update(location:pitch:direction:animated:)` method is frequently called to ensure that its visual appearance matches the map’s camera.
      */
-    @objc public var userCourseView: UIView? {
+    public var userCourseView: UserCourseView = UserPuckCourseView(frame: CGRect(origin: .zero, size: 75.0)) {
```

Comment:
The course view no longer appears on the map because of this change. `didSet` only gets called after the property is explicitly set, not after it is initialized to a default value. As a result, the course view is never added as a subview of the map view.

Reply:
Not sure if this is resolved. un-resolving until i'm positive.